SwiftUIHomeworks/SwiftUIHomeworks/ContentView.swift

```diff
@@ -7,20 +7,225 @@
+    static let typeVolumeTitle = "Объемы двигателя:"
+    static let availableTransmissionsTitle = "Доступные коробки передач:"
+    static let dateReleaseTitle = "Дата выпуска:"
+    static let priceTitle = "Цена:"
+    static let shareImageName = "square.and.arrow.up"
+    static let durationTime: TimeInterval = 0.5
+  }
+  
+  // MARK: - Public properties
+  
+  var body: some View {
+    NavigationView {
+      Form {
+        Section {
+          ZStack {
+            firstCarImage
+            secondCarImage
+          }
+          carsPicker
         }
-        .padding()
+        Section {
+          HStack {
+            Spacer()
+            modelView
+            Spacer()
+          }
+          bodyTypeView
+          releaseDateView
+          transmissionTypesView
+          enginePowerView
+          engineVolumeView
+        }
+        Section {
+          VStack {
+            HStack {
+              Text(Constants.changeComplectationTitle)
+              Spacer()
+              Text("\(carViewModel.cars[segmentedIndex].carClass[Int(sliderProgress)].rawValue)")
+            }
+          }
+          complectationSlider
+          priceView
+        }
+      }
+      .navigationTitle(carViewModel.getNameCar(index: segmentedIndex))
+      .toolbar {
+        shareButton
+      }
+    }
+  }
+  
+  // MARK: - Private properties
+  
+  @ObservedObject private var carViewModel = CarViewModel()
+  @State private var segmentedIndex = Constants.zeroValue
+  @State private var sliderProgress: Float = Float(Constants.zeroValue)
+  @State private var secondImageOffsetXValue = Constants.secondImageOffsetXValue
+  @State private var firstImageOffsetXValue = Constants.firstImageOffsetXValue
+  @State private var isSharedActionShown = false
+  
+  private var firstCarImage: some View {
+    Image(carViewModel.cars[segmentedIndex].carImageName.first ?? Constants.emptyString)
+      .resizable()
+      .scaledToFit()
+      .cornerRadius(Constants.cornerRadiusValue)
+      .offset(x: CGFloat(firstImageOffsetXValue), y: CGFloat(Constants.firstImageOffsetYValue))
+      .frame(width: Constants.firstImageWidthValue,
+             height: Constants.firstImageHeightValue)
+      .animation(.spring(response: Constants.responseValue, dampingFraction: Constants.dampingFractionValue, blendDuration: Constants.blendDurationValue))
+  }
+  
+  private var secondCarImage: some View {
+    Image(carViewModel.cars[segmentedIndex].carImageName.last ?? Constants.emptyString)
+      .resizable()
+      .scaledToFit()
+      .cornerRadius(Constants.cornerRadiusValue)
+      .offset(x: CGFloat(secondImageOffsetXValue), y: Constants.secondImageOffsetYValue)
+      .frame(width: Constants.secondImageWidthValue,
+             height: Constants.secondImageHeightValue)
+      .animation(.spring(response: Constants.responseValue, dampingFraction: Constants.dampingFractionValue, blendDuration: Constants.blendDurationValue))
+  }
+  
+  private var carsPicker: some View {
```

Comment:
```suggestion
  private var carsPickerView: some View {
```
  и везде также, где возвращаем some View